content/docs/0.8.1/deploy/upgrade/longhorn-manager.md

```diff
@@ -225,4 +240,4 @@ Steps to roll back:
     docker restart kubelet
     ```
 
-3. Rollback: Use `kubectl apply` or the Rancher catalog app to roll back Longhorn.
\ No newline at end of file
+3. Rollback: Use `kubectl apply` or the Rancher catalog app to roll back Longhorn.
```

Comment:
What is this about?

Reply:
I don't know why i local master branch is wrong. I have checked longhorn master, it should be step 4. I just corrected it.